File: 160a.py
n = int(input())
nums = list(map(int, input().split()))
nums.sort(reverse=True)
totsum = sum(nums)
me = []
for i in range(len(nums)):
    me.append(nums[i])
    if sum(me) > totsum - sum(me):
        break
print(len(me))

File: 160a.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    ios::sync_with_stdio(0);
    cin.tie(0);
    int n; cin >> n;
    vector<int> v(n);
    for (auto &i : v){cin >> i;}
    sort(v.begin(), v.end());
    reverse(v.begin(), v.end());
    int sum = accumulate(v.begin(), v.end(),0);
    int me = 0;
    int ans;
    for (int i=0; i<n; ++i){
        me+=v[i];
        ans++;
        if (me > sum-me) break;
    }
    cout << ans;
    
    
}